### PR TITLE
Cart item

### PR DESCRIPTION
**Pull request by spencersharpe-dev**
added new component and update quantity

### CODE REVIEW

**Review by tillyninjaspace — APPROVED** [review]
Looks good to me. Waiting on one more approval for merging.

**Review by zachgarrett24 — APPROVED** [review]
Looks good.

**Review by tillyninjaspace — CHANGES_REQUESTED** [review]
I want to make sure my code does not conflict or break your functionality so I'm gonna wait on merging this one!

**Pull request closed without merging**
